apps/server/src/modules/tldraw/tldraw.module.ts

```diff
@@ -0,0 +1,15 @@
+import { Module } from '@nestjs/common';
```

Comment:
We split consequent the modul and the "api" modul. Api is also not really perfect naming. WebSockets are also a public interface that can be used. In this view the websockets are also part of the api module. 
But for seperate deployments for rabbitMQ we use the naming format <name>-amqp.module.ts 

I will like to do it similar for this module. It also match the pattern of the controller.
tldraw.module.ts
tldraw.ws.ts that use the tldraw.module.ts

It look like overkill, but the argument on this place is the same like on others. More code will come and we do it similar to other modules.

Reply:
tldraw-ws.module.ts